src/pages/Acknowledgements/pageContent.tsx

```diff
@@ -153,9 +153,12 @@ export const pageContent = {
               do not proceed."
           />
         </Text>
+        <Text size="medium" className="mt20">
+          <FormattedMessage defaultMessage="We'll help you create a signing key and a withdrawal key for every validator you want to run. Signing and withdrawal keys are separate by design. Your signing key must be held online because it needs to be available at all times. Your withdrawal key doesn't need to be online so it's better to keep them separate for security." />
```

Comment:
```suggestion
          <FormattedMessage defaultMessage="We'll help you create a signing key and a withdrawal key for every validator you want to run. Signing and withdrawal keys are separate by design. Your signing key must be held online because it needs to be available at all times. Your withdrawal key doesn't need to be online so it's better to keep it separate for security." />
```